### PR TITLE
PPR api postgres migration setup: add type tables

### PR DESCRIPTION
**Pull request by doug-lovett**
Signed-off-by: Doug Lovett <doug@diamante.ca>

*Issue #:* /bcgov/entity#7495

*Description of changes:*
Add all missing type tables to the PPR API sqlalchemy model as a setup step for the migration to postgres.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-api/src/ppr_api/models/__init__.py [hunk 1]
```diff
@@ -13,11 +13,11 @@
 # limitations under the License.
 
 """This exports all of the models and schemas used by the application."""
-from .db import db  # noqa: I001
 from .address import Address
 from .client_code import ClientCode
 from .client_code_historical import ClientCodeHistorical
 from .court_order import CourtOrder
+from .db import db  # noqa: I001
```
Comment (thorwolpert) [inline ppr-api/src/ppr_api/models/__init__.py]: i've had issues with the db being imported in alphabetical order, as it's already imported by now in address, hence the 1001

File: ppr-api/src/ppr_api/models/type_tables.py [hunk 2]
```diff
@@ -18,17 +18,117 @@
...
+    # Relationships - Address
+    address = db.relationship('Address', back_populates='country_type')
+
+
+class ProvinceType(db.Model):  # pylint: disable=too-few-public-methods
+    """This class defines the model for the province_type table."""
+
+    __tablename__ = 'province_types'
+
+    province_type_cd = db.Column('province_type_cd', db.String(2), primary_key=True)
```
Comment (thorwolpert) [inline ppr-api/src/ppr_api/models/type_tables.py:40]: why **type**?

Comment (doug-lovett) [reply ppr-api/src/ppr_api/models/type_tables.py:40]: Do you mean the name of the file? The models in this file represent the type tables in the PPR data model.

File: ppr-api/src/ppr_api/models/__init__.py (hunk 1 above)
Comment (doug-lovett) [reply ppr-api/src/ppr_api/models/__init__.py]: Update checked in with order change and disable of warning removed.

**Review by thorwolpert — APPROVED** [review]

**Pull request merged**